executor/show_test.go

```diff
@@ -329,8 +329,13 @@ func (s *testSuite2) TestShowCreateTable(c *C) {
 	tk.MustExec("create table t1(a int,b int)")
 	tk.MustExec("drop view if exists v1")
 	tk.MustExec("create or replace definer=`root`@`127.0.0.1` view v1 as select * from t1")
```

Comment:
try `create view definer=`root`@`127.0.0.1` view v1 as select * from t1 where a + 1 < "123"`

Reply:
correct grammar stmt should be `create definer=root@127.0.0.1 view v1 as select * from t1 where a + 1 < "123"`

Reply:
Yep, what I meant was that change the origin sql to 
```
create definer=root@127.0.0.1 view v1 as select * from t1 where a + 1 < "123"
```
and check the results.
And remember to check whether the result is compatible with MySQL.